src/librustdoc/html/render/mod.rs

```diff
@@ -95,7 +95,31 @@ crate struct IndexItem {
 /// A type used for the search index.
 #[derive(Debug)]
 crate struct RenderType {
+    ty: Option<DefId>,
+    idx: Option<usize>,
```

Comment:
I can only find `idx` set to `None`. Am I missing something maybe? Otherwise please remove this field.